### PR TITLE
Renamed "Web platform" TOC bucket to "Development tips for Microsoft Edge"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered articles for review:
* Topmost article, see 'web platform' h2 section renamed "devmt tips":
   * https://review.learn.microsoft.com/en-us/microsoft-edge/develop-web-microsoft-edge?branch=pr-en-us-2400#development-tips-for-microsoft-edge
* The top-level article for 'web platform' TOC bucket, retitled:
   * https://review.learn.microsoft.com/en-us/microsoft-edge/web-platform/web-platform?branch=pr-en-us-2400
* Landing card title: 
   * https://review.learn.microsoft.com/en-us/microsoft-edge/developer/index?branch=pr-en-us-2400
* Hub card title:
   * https://review.learn.microsoft.com/en-us/microsoft-edge/index?branch=pr-en-us-2400

GitHub rendered version of the above:
* https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/webplat-toc/microsoft-edge/develop-web-microsoft-edge.md#development-tips-for-microsoft-edge
* https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/webplat-toc/microsoft-edge/web-platform/web-platform.md
* Landing: Line 177: "Card r2c3" "title: Development tips for Microsoft Edge"  https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/webplat-toc/microsoft-edge/developer/index.yml
* Hub: Line 308: card title "Development tips for Microsoft Edge"  https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/webplat-toc/microsoft-edge/index.yml

### CODE REVIEW

File: microsoft-edge/develop-web-microsoft-edge.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ See [Test and automation in Microsoft Edge](test-and-automation/test-and-automat
 
 
 <!-- ====================================================================== -->
-## Web platform
+## Development tips for Microsoft Edge
 
 Considerations for developing websites and products for the web platform include the following:
```
Comment (Reezaali) [inline microsoft-edge/develop-web-microsoft-edge.md:106]:
```suggestion
There are many tips and tricks to be aware of when building on the web platform, which include: 
```

Comment (mikehoffms) [reply microsoft-edge/develop-web-microsoft-edge.md:106]: fixed locally, trying (possibly more natural/transparent):
There are many tips and tricks to be aware of when building on the web platform, including:

File: microsoft-edge/develop-web-microsoft-edge.md [hunk 2]
```diff
@@ -101,9 +101,9 @@ See [Test and automation in Microsoft Edge](test-and-automation/test-and-automat
 
 
 <!-- ====================================================================== -->
-## Web platform
+## Development tips for Microsoft Edge
 
-Considerations for developing websites and products for the web platform include the following:
+There are many tips and tricks to be aware of when building on the web platform, including:
```
Comment (captainbrosset) [inline microsoft-edge/develop-web-microsoft-edge.md:106]: I think we can simplify `web platform` to `web`, and we somehow need to say that those tips are specific to Edge only. Suggestion:

```suggestion
There are many tips and tricks to be aware of when building web sites or apps that work in Microsoft Edge, including:
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

**Comment by captainbrosset** [comment]
We should also think about renaming the "Web Platform" card on the landing page: https://learn.microsoft.com/en-us/microsoft-edge/developer/

File: microsoft-edge/develop-web-microsoft-edge.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/develop-web-microsoft-edge.md:106]: in this top-level doc and the bucket intro:
There are many tips and tricks to be aware of when building websites or web apps that work in Microsoft Edge, including:

**Review by captainbrosset — APPROVED** [review]

File: microsoft-edge/develop-web-microsoft-edge.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ ms.author: msedgedevrel
 ms.topic: conceptual
```
Comment (mikehoffms) [inline microsoft-edge/develop-web-microsoft-edge.md:6]: CapB wrote on main page of PR:

We should also think about renaming the "Web Platform" card on the landing page: https://learn.microsoft.com/en-us/microsoft-edge/developer/

File: microsoft-edge/web-platform/web-platform.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ ms.date: 11/28/2022
 ---
 # Web platform overview
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/web-platform.md:11]: @captainbrosset @Reezaali - change page title from "Web platform overview" to be similar to new toc bucket name? that's:
Development tips for Microsoft Edge
https://learn.microsoft.com/en-us/microsoft-edge/web-platform/web-platform

File: microsoft-edge/develop-web-microsoft-edge.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/develop-web-microsoft-edge.md:6]: pushed fix

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/web-platform/web-platform.md (hunk 4 above)
Comment (captainbrosset) [reply microsoft-edge/web-platform/web-platform.md:11]: Yes I do think we should change the page title too.

Comment (mikehoffms) [reply microsoft-edge/web-platform/web-platform.md]: retitled globally same as bucket name

**Pull request merged**